src/SignalRClient/signalrclient.yml

```diff
@@ -5,9 +5,8 @@
       branchOrCommit: master
       project: src/SignalRClient/SignalRClient.csproj
     readyStateText: SignalR Client
-    sdkVersion: 3.1.201
-    aspNetCoreVersion: '3.1'
-    runtimeVersion: '3.1'
+    sdkVersion: 5.0.100-preview.2.20120.3
```

Comment:
If it works with the latest sdk, and I don't see any reason why not, then you can omit this argument as by default it will use the latest 5.0.*